.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,20 @@
 <!--
- Change Github PR Title 
-
- Your title must be in the following format: 
- - `topic(Area): Feature`
+ Title format: 
+ - `Topic(Area): Feature`
  - `Topic` must be one of `build|ci|docs|feat|fix|perf|refactor|chore|test`
 
  Sample Titles:
```

Comment:
we don't need samples now I think. These are the areas that I am thinking:
acl|audit|backup|badger|cdc|dql|export|graphql|indexing|multi-tenancy|raft|restore|upgrade